src/calculateRentalCost.js

```diff
@@ -28,7 +28,16 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const costEveryDay = 40;
+  let totalcost = costEveryDay * days;
```

Comment:
Rename 'totalcost' variable to 'totalCost' to follow camelCase naming convention.